ledger/accountdb.go

```diff
@@ -84,9 +84,10 @@ var accountsSchema = []string{
 // TODO: Post applications, rename assetcreators -> creatables and rename
 // 'asset' column -> 'creatable'
 var creatablesMigration = []string{
-	`ALTER TABLE assetcreators ADD COLUMN ctype INTEGER DEFAULT 0`,
+	`ALTER TABLE assetcreators ADD COLUMN ctype integer DEFAULT 0`,
```

Comment:
any particular reason you're changing the `INTEGER`->`integer` ?  this is a sqlite language, which we currently keep in capital letters.

Reply:
I have no idea but fixed